java/src/main/native/ai_onnxruntime_OnnxTensor.c

```diff
@@ -66,28 +64,32 @@
  */
 JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OnnxTensor_createTensorFromBuffer
         (JNIEnv * jniEnv, jclass jobj, jlong apiHandle, jlong allocatorHandle, jobject buffer, jint bufferPos, jlong bufferSize, jlongArray shape, jint onnxTypeJava) {
-    (void) jobj; // Required JNI parameter not needed by functions which don't need to access their host object.
+    (void) jobj;  // Required JNI parameter not needed by functions which don't need to access their host object.
     const OrtApi* api = (const OrtApi*) apiHandle;
     OrtAllocator* allocator = (OrtAllocator*) allocatorHandle;
     const OrtMemoryInfo* allocatorInfo;
-    checkOrtStatus(jniEnv, api, api->AllocatorGetInfo(allocator,&allocatorInfo));
+    OrtErrorCode code = checkOrtStatus(jniEnv, api, api->AllocatorGetInfo(allocator, &allocatorInfo));
+    if (code != ORT_OK) {
+      return (jlong) NULL;
+    }
 
     // Convert type to ONNX C enum
     ONNXTensorElementDataType onnxType = convertToONNXDataFormat(onnxTypeJava);
 
     // Extract the buffer
-    char* bufferArr = (char*)(*jniEnv)->GetDirectBufferAddress(jniEnv,buffer);
+    char* bufferArr = (char*)(*jniEnv)->GetDirectBufferAddress(jniEnv, buffer);
     // Increment by bufferPos bytes
     bufferArr = bufferArr + bufferPos;
 
     // Extract the shape information
     jboolean mkCopy;
-    jlong* shapeArr = (*jniEnv)->GetLongArrayElements(jniEnv,shape,&mkCopy);
-    jsize shapeLen = (*jniEnv)->GetArrayLength(jniEnv,shape);
+    jlong* shapeArr = (*jniEnv)->GetLongArrayElements(jniEnv, shape, &mkCopy);
+    jsize shapeLen = (*jniEnv)->GetArrayLength(jniEnv, shape);
 
     // Create the OrtValue
-    OrtValue* ortValue;
-    checkOrtStatus(jniEnv, api, api->CreateTensorWithDataAsOrtValue(allocatorInfo,bufferArr,bufferSize,(int64_t*)shapeArr,shapeLen,onnxType,&ortValue));
+    OrtValue* ortValue = NULL;
+    checkOrtStatus(jniEnv, api, api->CreateTensorWithDataAsOrtValue(allocatorInfo, bufferArr, bufferSize,
```

Comment:
>checkOrtStatus [](http://example.com/codeflow?start=4&length=14)

The return code is not checked. Should we release OrtValue and return NUNLL as above?